src/index.js

```diff
@@ -0,0 +1,73 @@
+// import style from style.css file (required)
+import './style.css';
+
+// import the function used to populate HTML for each task
+import populateHtmlForEachTask from '../modules/populateHtmlForEachTask.js';
+
+// Import the tasksList Class
+import TasksList from '../modules/tasksListClass.js';
+
+const tasks = new TasksList();
+
+/* Call the populateHtmlForEachTask function to populate
+the HTML list item element for each task in the local storage */
+const storedTasks = JSON.parse(localStorage.getItem('storedTasks')) || [];
+populateHtmlForEachTask(storedTasks);
```

Comment:
What do you think about declaring `storedTasks `variable inside `'../modules/populateHtmlForEachTask.js'` and just call populateHtmlForEachTask();
I think it would be mantain the index.js file cleaner.

Reply:
Thank you for your feedback!
The storedTasks variable is a global variable, that is why I declared it outside the function.